src/main/java/com/aws/greengrass/logmanager/CloudWatchLogsUploader.java

```diff
@@ -134,7 +136,8 @@ private boolean uploadLogs(String logGroupName, String logStreamName, List<Input
             .build();
 
         try {
-            PutLogEventsResponse putLogEventsResponse = this.cloudWatchLogsClient.putLogEvents(request);
+            PutLogEventsResponse putLogEventsResponse = cloudWatchWrapper
+                    .execute(client -> client.putLogEvents(request));
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

`software.amazon.awssdk.services.cloudwatchlogs.CloudWatchLogsClient.putLogEvents` API can also throw the following exception types: `InvalidParameterException`, `InvalidSequenceTokenException`, `UnrecognizedClientException`, `SdkClientException`, `SdkException`, `AwsServiceException`, `DataAlreadyAcceptedException`, `ServiceUnavailableException`, `ResourceNotFoundException`, `CloudWatchLogsException`, `UnsupportedOperationException`. We recommend handling these uncaught exceptions as well.